src/test/run-tests.sh

```diff
@@ -3,7 +3,9 @@
 SCRIPTDIR="$(dirname ""$0"")"
 
 MOZIOT_HOME="${MOZIOT_HOME:=${HOME}/.mozilla-iot}"
+MOZIOT_HOME="${MOZIOT_HOME}/test"
 if [ ! -f "${MOZIOT_HOME}/ssl/certificate.pem" ]; then
```

Comment:
Having the export inside the if feels wrong.

Reply:
I agree, but it also seemed wrong to export unnecessarily.